### PR TITLE
Unsubscribing emails

### PR DESCRIPTION
**Pull request by otlaitil**
For https://www.pivotaltracker.com/story/show/94487906

Adds a feature for petition signer to unsubscribe from future updates on the petition that he/she has signed.

### CODE REVIEW

File: app/views/petition_mailer/notify_signer_of_threshold_response.html.erb [hunk 1]
```diff
@@ -9,3 +9,5 @@
 <p>Thanks,</p>
 
 <p>HM Government e-petitions <%= link_to home_url, home_url %></p>
+
+<p>Don't want to receive any more emails regarding this petition? Unsubscribe by clicking <%= link_to 'unsubscribe', unsubscribe_signature_url(@signature, @signature.unsubscribe_token) %></p>
```
Comment (otlaitil) [inline app/views/petition_mailer/notify_signer_of_threshold_response.html.erb]: @yahoopete this needs copy

File: features/suzie_unsubscribes_from_petition_updates.feature [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Feature: Unsubscribing from petiton updates as Suzie
```
Comment (otlaitil) [inline features/suzie_unsubscribes_from_petition_updates.feature]: Would appreciate some feedback on this cucumber test... I tested only the success case (see the commit message on the commit this test was introduced).

Comment (h-lame) [inline features/suzie_unsubscribes_from_petition_updates.feature]: I think it's good.

**Comment by otlaitil** [comment]
Thank you @mybits and @h-lame for the PR comments! I pushed some fixes. :balloon:

**Pull request merged**